Products/DataCollector/zing/fact.py

```diff
@@ -8,6 +8,15 @@
 
 import time
 
+
+class FactKeys(object):
+    CONTEXT_UUID_KEY = "contextUUID"
+    META_TYPE_KEY = "meta_type"
+    NAME_KEY = "name"
+    MEM_CAPACITY_KEY = "mem_capacity"
```

Comment:
This particular one seems rather type specific? The others are good generally applicable ones. Just curious why this got picked up here...

Reply:
This comes from the initial implementation we had: https://github.com/zenoss/ZenPacks.zenoss.ZingConnector/blob/develop/ZenPacks/zenoss/ZingConnector/fact.py#L91-L122
it's device specific information that we can reliably extract